### ISSUE TITLE
Problem reading photo's

### ISSUE THREAD

**Issue by fretje** [perevoznyk/swelio-sdk#15]
For an old project, I upgraded to the latest version of swelio sdk recently as some newer ID cards couldn't be read anymore.

There was one breaking change apparently.

I was using `Card.ReadPhotoBytes()`, and the `byte[]` returned from this method was just put on the screen in a WPF app by simply binding an `ImageBrush` straight to that `byte[]` value, and the picture showed up nice.

But this method was removed apparently. What there is now is a `EncodePhoto()` method. The returnvalue is also a `byte[]`... but when I bind that `byte[]` to the `ImageBrush`, the image doesn't show up.

When I try to read that byte[] into an image:

```c#
            var photoBytes = card.EncodePhoto();
            var image = Image.FromStream(new MemoryStream(photoBytes));
```

An `ArgumentException "Parameter is not valid."` is thrown from inside `System.Drawing`.

As the code in the ReadPhoto method is closer to what used to be in ReadPhotoBytes... (the only difference is that it's wrapped into a Bitmap), I tried to do this:

```c#
            var image = card.ReadPhoto();
            using (var stream = new MemoryStream())
            {
                image.Save(stream, image.RawFormat);
                var bytes = stream.ToArray();
            }
```

But that `Save` call throws a `System.Runtime.InteropServices.ExternalException "A generic error occurred in GDI+."` also from `System.Drawing`.

That lead me to [this SO answer from the magnificent Jon Skeet](https://stackoverflow.com/a/336396/101371), which says that you shouldn't dispose the MemoryStream that you used to construct the Bitmap (the bitmap itself is responsible for that) I think if that would be fixed that I can keep the changes to my code minimal by using this latter method to get to the original byte[] that my app is expecting...

What do you think? Shall I make a PR?

**Comment by perevoznyk** [comment]
Fixed in release 1.18.17 - added new method ReadPhotoBytes to Card class and removed stream disposal from ReadPhoto method as suggested

**Comment by fretje** [comment]
Very nice... thanks for the swift response!
